### PR TITLE
Stop tool tips from resizing page and add padding

### LINKED ISSUE #112: Calendar access link needs padding
It looks bad at the very bottom of the page. More importantly the tooltips open below the link and cause the page to become temporarily larger which moves your mouse off the tooltip until the page shrinks. This causes a maddening cycle.

jun-14-2018 13-33-48

### PR DESCRIPTION
**Pull request by jforgue**
Closes #112.

Screenshots:
image
image
image

### CODE REVIEW

File: app/views/rosters/index.haml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 %h1.title Rosters
 
-%table.table.auto-width.col-xs-offset-3
+%table.table.auto-width.col-xs-offset-1
```
Comment (werebus) [inline app/views/rosters/index.haml:3]: This looks worse, but it was a little off anyways.

I don't think we should use an offset that is a fixed number of columns to make up for the table not being  the full width.

The `.table` class actuall does make the "`width`" `100%`, but all the columns have so little information in them that they don't fill up the table.

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -22,29 +22,33 @@ $navbar-default-link-active-color: #ddd;
...
   background-color: #fafafa;
-  margin: 0 auto;
-  min-width: 50em;
-  max-width: 70em;
+  min-width: 70em;
+  font-family: sans-serif;
 }
 
 .center {
   text-align: center;
```
[the remark this answers is not in the document]
Comment (jforgue) [reply app/assets/stylesheets/application.scss:45]: Oh, that's right

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -22,29 +22,33 @@ $navbar-default-link-active-color: #ddd;
...
+  flex-direction: column;
+  align-items: center;
 }
 
 #main-content {
   background-color: #fafafa;
-  margin: 0 auto;
-  min-width: 50em;
-  max-width: 70em;
+  min-width: 70em;
```
[the remark this answers is not in the document]
Comment (jforgue) [reply app/assets/stylesheets/application.scss:40]: It seemed to want to stay at the minimum after my changes which made it smaller than the current style and the calendar doesn't really collapse much regardless (I should probably fix that at some point).

File: app/assets/stylesheets/application.scss [hunk 4]
```diff
@@ -22,29 +22,33 @@ $navbar-default-link-active-color: #ddd;
...
+  font-family: sans-serif;
 }
 
 .center {
   text-align: center;
 }
 
+.pad-bottom {
+  padding-bottom: 1%;
```
[the remark this answers is not in the document]
Comment (jforgue) [reply app/assets/stylesheets/application.scss:49]: So it would seem

**Pull request merged**